indigo/bulk_creator.py

```diff
@@ -819,10 +833,14 @@ def link_amendment_passive(self, row):
         if not amending_work:
             return self.create_task(row.work, row, task_type='link-amendment-passive')
 
-
         if self.dry_run:
             row.relationships.append(f'Amended by {amending_work}')
-            row.notes.append("An 'Apply amendment' task will be created on this work")
+            note = "An 'Apply amendment' task will be created on this work"
+            if self.cancel_amendment_tasks:
+                note += ' and CANCELLED'
```

Comment:
Why not simply not create the task? So skip, rather than cancel?

Reply:
We'd rather have a record of the task having been created in the system, so that it can be discussed, reopened if needed, or just commented on.